mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/quota/CassandraCurrentQuotaManager.java

```diff
@@ -89,16 +93,16 @@ public QuotaCountUsage getCurrentMessageCount(QuotaRoot quotaRoot) throws Mailbo
     }
 
     @Override
-    public QuotaSizeUsage getCurrentStorage(QuotaRoot quotaRoot) throws MailboxException {
+    public QuotaSizeUsage getCurrentStorage(QuotaRoot quotaRoot) {
         ResultSet resultSet = session.execute(getCurrentStorageStatement.bind(quotaRoot.getValue()));
         if (resultSet.isExhausted()) {
             return QuotaSizeUsage.size(0L);
         }
         return QuotaSizeUsage.size(resultSet.one().getLong(CassandraCurrentQuota.STORAGE));
     }
```

Comment:
Can we have a `public CurrentQuota getCurrentStorage(QuotaRoot quotaRoot)` as well and try to use it in most places to save one counter read?

Reply:
Actually when I look at the code, we really use `getCurrentMessageCount` and `getCurrentStorage` only in `StoreQuotaManager` in separated methods. So I think this is not needed for now (but I guess I will need it anyway for the recomputing current quota task, but will figure out in next PR)